app/views/gsa18f/procurements/form.html.erb

```diff
@@ -89,5 +89,8 @@
       <%= f.text_area :additional_info, class: 'form-control' %>
     </div>
     <%= f.submit "Submit for approval", class: 'btn btn-default' %>
+    <% if !@procurement.id.nil? %>
```

Comment:
Might be cleaner with [`persisted?`](http://apidock.com/rails/v4.2.1/ActiveRecord/Persistence/persisted%3F)